tips-tricks/TroubleshootingandTips.rst

```diff
@@ -669,7 +669,7 @@ Notice the double escape required!
 .. |image1| image:: ../media/kylo-troubleshooting/2_executesparkjob.png
 
 Configuration on a Node with Small Root Filesystem
-========================================================================
+=======
```

Comment:
This is generating a warning message. Please ensure that the underlines are the same length as the title.

    tips-tricks/TroubleshootingandTips.rst:672: WARNING: Title underline too short.